extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -604,6 +625,9 @@ export class AmpAnalytics extends AMP.BaseElement {
    * @private
    */
   handleEvent_(trigger, event) {
+    if (trigger?.session?.['persistEvent'] && this.sessionManager_) {
```

Comment:
What's the difference between this and the other key?

Reply:
`persistSessionEvent` is created to let `amp-analytics` know if we should initialize SessionManager. `persistEvent` is to know what trigger's should act as events that we want to keep the timestamp of.